oneview_redfish_toolkit/api/scmb.py

```diff
@@ -61,14 +66,37 @@ def _scmb_key_path():
     return os.path.join(_scmb_base_dir(), SCMB_KEY_NAME)
 
 
-def check_cert_exist():
+def init_event_service():
+    if config.auth_mode_is_conf():
+        # Loading scmb connection
+        if has_valid_certificates():
+            logging.info('SCMB certs already exists and is valid...')
+        else:
+            logging.info('SCMB certs not found. '
+                         'Checking if it was already generated in Oneview...')
+            get_cert()
+            logging.info('Got certs. Testing connection...')
+            if not is_cert_working_with_scmb():
+                logging.error('Failed to connect to scmb. Aborting...')
+                exit(1)
+        scmb_thread = threading.Thread(target=listen_scmb)
+        scmb_thread.daemon = True
+        scmb_thread.start()
+    else:
+        logging.warning("Authentication mode set to session. SCMB events will "
+                        "be disabled")
+
+
+def has_valid_certificates():
```

Comment:
This method is a internal method? We could put it as private method: `_has_valid_certificates()`